public/index.html

```diff
@@ -278,6 +278,11 @@
             <div class="project-name">Url Shortener</div>
             <a href="https://github.com/dhairyagothi/50_days_50_web_project/tree/Main/public/url_shortener" class="demo-link">Demo</a>
         </div>
+        <div class="table">
+            <div class="day-number">Day 50</div>
+            <div class="project-name">E-Commerce Website with Simple Cart Functionality</div>
```

Comment:
add it as DAY 51 I recently merged a PR 
PULL the mainn branch

Reply:
Will do in some time